source/js/main.js

```diff
@@ -1,11 +1,24 @@
 require("core-js-bundle");
-window.globalVariables = require("./global-variables");
-window.cookie = require("./cookie");
-window.cookieAlert = require("./cookie-alert");
-window.errorHandling = require("./error-handling");
-window.headerNavigation = require("./header-navigation");
-window.googleAds = require("./google-ads");
+// Utils
+
+// Components
+
+// Init Utils
+
+// Init Components
+window.globalVariables = require("./global-variables"); // helper
+window.cookie = require("./cookie");                    // component
+window.cookieAlert = require("./cookie-alert");                    // component
+window.errorHandling = require("./error-handling");// helper
+window.headerNavigation = require("./header-navigation");                    // component
+window.googleAds = require("./google-ads");   
+window.cpl = require("./characters-per-line");// helper
+window.textEllipsis = require("./text-ellipsis");// helper                 // component
+window.teaser = require("./teaser");                    // component
```

Comment:
Is all that space between the comments necessary?

Reply:
Nope. But it's work in progress.
So, please have a look into the logic. We can talk about code styling later.